doc/source/configuration/host-images.rst

```diff
@@ -14,6 +14,8 @@ When configured, an image will be downloaded to the seed during the
 ``kayobe seed service deploy`` step, and subsequently deployed using bifrost
 with ``kayobe overcloud provision``.
 
+FIXME: Should we do this by default in Zed+?
```

Comment:
Might be a bit redundant. Everyone has to migrate off cs8/rl8/focal on yoga anyway so should be using the images by then

Reply:
But then it would make sense for new deployments to use them so perhaps it is worth it.